tests/lib/cascading-config-array-factory.js

```diff
@@ -189,15 +186,15 @@ describe("CascadingConfigArrayFactory", () => {
 
                     await prepare();
                     cwd = getPath();
-                    
+
                     factory = new CascadingConfigArrayFactory({ cwd });
                 });
 
                 afterEach(() => cleanup());
 
                 // no warning.
                 describe("when it lints 'subdir/exist-with-root/test.js'", () => {
-                    beforeEach(async () => {
+                    beforeEach(async() => {
```

Comment:
The actual formatting will be valid per https://github.com/eslint/eslint/commit/472025f2814d0360fe8d4cddbcba049982e1cd43 but that change hasn't been released yet, so let's better release and upgrade `eslint-config-eslint` first.